src/test-interface.js

```diff
@@ -21,3 +22,7 @@ window.accessLocalForageStore = function (store_name) {
         );
     });
 };
+
+window.throwError = function () {
+    return new CriticalException();
```

Comment:
```suggestion
    throw new CriticalException();
```

Also, it would be good if you passed the parameters (`message`, `context`, `enduser_message` and the actual error obj) and then checked whether they get handled correctly.